daffodil-cli/LICENSE-bin

```diff
@@ -306,9 +318,23 @@ subcomponents is subject to the terms and conditions of the following licenses.
     permission. Title to copyright in this work will at all times remain with
     copyright holders.
 
-  This product bundles source from the Open Grid Forum Data Format Description
-  Language (DFDL) v1.0 Specification. The source is available under the following
-  license:
+  This product bundles content from the Open Grid Forum Data Format Description
+  Language (DFDL) v1.0 Specification, including the following files:
+    - daffodil-test/src/test/resources/org/apache/daffodil/ibm-tests/dpaspc7132.dfdl.xsd
```

Comment:
We don't typically see test files bundled in the binary release.  Do you in fact ship tests in your binary?

Reply:
Tests aren't bundled in the binary helpers, will remove.